src/features/profile/pages/NewEmailSelection/NewEmailSelection.web.test.tsx

```diff
@@ -18,7 +18,7 @@ describe('<NewEmailSelection />', () => {
     it('should not have basic accessibility issues', async () => {
       const { container } = render(reactQueryProviderHOC(<NewEmailSelection />))
 
-      await waitFor(async () => {
+      await act(async () => {
```

Comment:
c'est sémantiquement bizarre de wrapper un `expect` dans un `act`

Reply:
Je crois qu'Inès a fixe la flakiness du test en faisant cela. Le ticket a déjà pris beaucoup trop de temps pour que je creuse là dessus , tu préfères que j'annule le commit ?

Reply:
je préfère garder des tests stables

si annuler le commit = rendre le test flaky, alors non

ça me parait bizarre que ce truc stabilise un test